### DISCUSSION TITLE
Manage all the shared links

### DISCUSSION THREAD

**Discussion by racecrew** [photoprism/photoprism#3690] [Ideas]
I would like to view/update all the shared links from one place. For example, to remove all of them at once. Is there a feature like this? I didn't find.

**Comment by graciousgrey** [comment]
Many improvements are planned for sharing:

https://github.com/photoprism/photoprism/issues?q=is%3Aissue+is%3Aopen+Sharing+in%3Atitle
- https://github.com/photoprism/photoprism/issues/1330
